float64[5] angles
float64[3] commands
---
float64[5] velocities
bool singularMatrix